cookbooks/bach_deployment/attributes/default.rb

```diff
@@ -0,0 +1,12 @@
+force_default['bach']['deploy']['appfile']['data'] = {
+  appfoo: {
+    repo_url: get_binary_server_url() 
```

Comment:
This is an implicit dependency to the `bcpc` cookbook .  Let's make this to some default URL as well.